wazuh/wazuh-kibana-app
# Fix visualizations with dark mode background in PDF report

Hi team,

this fixes the dark background in some visualizations when generating a PDF report.

Closes #3224 

Test:

- Go to Security Events
- Generate a PDF report
- Check there's no dark background in visualizations
